components/webpack/webpack.config.js

```diff
@@ -48,8 +48,8 @@ module.exports = (env, argv) => ({
       },
       // Loads font files for Font Awesome
       {
-        test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
-        loader: 'url-loader?limit=13000&minetype=application/font-woff'
+        test: /\.woff(2)|m4a|mp4?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
```

Comment:
just for my own knowledge: what this file is about? :)

Reply:
Well, I don't know why this change is added. (Maybe added by other devs during the development)
Deleted.